SparkPost/matchbox
# FE-913 Move Global Styles to Styled Components

### What Changed
– Global styles are no longer bundled into a styles.css export
– Global styles are generated through ThemeProvider

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Verify global styles are working in any story

### PR Checklist
